packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.tsx

```diff
@@ -411,14 +361,6 @@ export class BaseFloatingPicker<T, P extends IBaseFloatingPickerProps<T>> extend
     }
   }
 
-  private _getTextFromItem(item: T, currentValue?: string): string {
-    if (this.props.getTextFromItem) {
```

Comment:
Why remove this? How are you getting the value for the autofill? If you don't want that functionality then the autofill should get changed to a regular textbox.

Reply:
We aren't using the value suggested display value for the autofill, but autofill has some public methods dealing with the input that we are leveraging. Let me know if you feel strongly, and I can duplicate these methods out and change the autofill in this component to be an input type

Reply:
That's fair, it's just heavy weight for what you need. Is there any reason why you aren't using the display value? Could there be changes that would allow you to use it?

Reply:
For my use of the component, we just don't need it. It has never actually been used in the history of this component (so this isn't a breaking change). I suppose we could, in the future, allow to use the display value.